libs/@local/hash-graph-client/python/graph_client/models.py

```diff
@@ -105,6 +105,7 @@ class EntityTypeQueryToken(Enum):
     label_property = "labelProperty"
     links = "links"
     inherits_from = "inheritsFrom"
+    children = "children"
```

Comment:
We'll also need to update the filter; I'll do that in a follow-up PR. I wrote the code for it but forgot to add it to the codegen pipeline 🤦